setup.py

```diff
@@ -26,7 +26,7 @@
     "codecov>=2.0.22",
     "docutils>=0.10,<0.16",
     "flake8>=3.7.7",
-    "napari>=0.2.10",
+    "napari[pyqt5]>=0.2.10",
```

Comment:
Are we waiting to remove this because it's an interface change and necessitates a 4.x?

Reply:
Yeah. It was already removed on branch `release-4.0` or w.e. that branch is called